src/Components/Components/test/RendererTest.cs

```diff
@@ -730,6 +730,160 @@ public async Task CanAsyncDispatchEventsToNestedComponents()
             Assert.Equal(2, state);
         }
 
+        // The delegate target points to the outer component
+        [Fact]
+        public async Task EventDispatching_DelegateParameter_MethodToDelegateConversion()
```

Comment:
Is the point of this test to show that it's unnecessary to use `CreateDelegate` if `action.Target` already points to the component you want to be notified after the event?

If so that's great, but maybe add a comment to that effect, as it's pretty subtle!

Reply:
I've added comments to all of these new tests.